### ISSUE TITLE
[Feature Request] Change labels for options in PieMenu

### ISSUE THREAD

**Issue by solajim** [Grubuntu/PieMenu#112]
If possible, would you be willing to let us choose labels for each shortcut?

I've included a screenshot below to explain:
![Sans titre](https://github.com/user-attachments/assets/4991081e-c8fe-470f-9db8-7e44922cc2de)

**Comment by Grubuntu** [comment]
Yes, In the next version, I'm going to remove the ‘PieMenu’ header, which will no longer be necessary as I've included a possible sort by WB :

![Capture d’écran (271)](https://github.com/user-attachments/assets/f076abf3-afb2-4919-b012-52d7e562dd4d)

**Comment by Grubuntu** [comment]
‘PieMenu_’ header has been removed from nested PieMenus in V1.10
![Piemenus](https://github.com/user-attachments/assets/6fa85df2-0c16-42b7-bbbc-0652a3b476d0)
